src/Symfony/Component/Serializer/Normalizer/DenormalizerInterface.php

```diff
@@ -27,6 +27,13 @@
      * @param array  $context options available to the denormalizer
      *
      * @return object
+     *
+     * @throws \Symfony\Component\Serializer\Exception\BadMethodCallException
+     * @throws \Symfony\Component\Serializer\Exception\InvalidArgumentException
+     * @throws \Symfony\Component\Serializer\Exception\UnexpectedValueException
+     * @throws \Symfony\Component\Serializer\Exception\ExtraAttributesException
+     * @throws \Symfony\Component\Serializer\Exception\LogicException
+     * @throws \Symfony\Component\Serializer\Exception\RuntimeException
```

Comment:
>you can't know when an interface throw an exception or not

Interfaces don't throw exceptions, their implementations do. I disagree with putting them into interface, because when you make different implementation which don't throw those exceptions, you are not respecting the interface. Interface specifies what class implementing it MUST implement. I don't see a need here to force the implementation to use these exceptions